tests/test_traceview.py

```diff
@@ -19,6 +19,9 @@
 
 TV_API_KEY = os.environ.get("TV_API_KEY", None)
 
+# The total_requests API requires a non-default app to measure
+TV_REQUESTS_APP = os.environ.get("TV_REQUESTS_APP", None)
```

Comment:
@mcodd after thinking about this a bit more, what are your thoughts on renaming this to `TV_APP_NAME`? i was thinking that it should be usable by other tests in the future, so encourage reuse by naming it something a bit more accessible and less restricted?

thoughts?